apps/server/src/shared/domain/entity/submission.entity.spec.ts

```diff
@@ -26,15 +26,37 @@ describe('Submission entity', () => {
 	});
 
 	describe('constructor is called', () => {
-		describe('when files are passed', () => {
-			it('create with studentFiles should be possible', () => {
+		describe('when studentFiles are passed', () => {
```

Comment:
Is this test (and the one below) really necessary? Doesn't it only test if the factory works?

Reply:
We need it, because we have logic in the constructor

Reply:
But we will remove the key from the collection after file migration

Reply:
But if you want to test the constructor, you should call the constructor in the test itself and not the factory in setup.

Reply:
You still call the constructor in setup. I would say, you should call it in the test itself. If you think about it in the Arrange-Act-Assert pattern, our setup corresponds to Arrange. The call of the method that is tested - the Act - should be in the test itself, meaning in the it-block.